### PR TITLE
Refresh Flatpak tutorial with Filezilla examples

### PR DESCRIPTION
**Pull request by krisdale**
- Update examples to use Filezilla
- Update screenshots for Gnome Software
- Add details re command line snippets available in Clear Linux Store
- Condense and update Prereq.

Signed-off-by: Kristal Dale

### CODE REVIEW

File: source/clear-linux/tutorials/flatpak/flatpak.rst [hunk 1]
```diff
@@ -1,103 +1,127 @@
...
-our :ref:`bare metal installation guide <bare-metal-install-desktop>`.
+* |CL| installed on host system
 
-Install Flatpak on your host system
-===================================
+  Refer to :ref:`Get started <get-started>` for installation instructions.
 
-Flatpak is included as part of the bundle `desktop`. To install the
-application, log in to your user account and enter the following command:
+* `desktop` bundle installed
```
Comment (mvincerx) [inline source/clear-linux/tutorials/flatpak/flatpak.rst]: `desktop-autostart`.

File: source/clear-linux/tutorials/flatpak/flatpak.rst [hunk 2]
```diff
@@ -1,103 +1,127 @@
...
 
-.. code-block:: bash
+  Flatpak is included as part of the `desktop` bundle. The Flathub repository is
```
Comment (mvincerx) [inline source/clear-linux/tutorials/flatpak/flatpak.rst]: Please revise this per @bktan8 comments in #481  
Add keyword to close that issue too.

**Comment by krisdale** [comment]
Reopening to see if I can make the needed amendment to commit signature.

**Comment by krisdale** [comment]
Reopening as updates are valid and will correct commit message.

File: source/clear-linux/tutorials/flatpak/flatpak.rst (hunk 1 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: Hold pls @krisdale . @bktan8 please first respond to my comments on #481.

File: source/clear-linux/tutorials/flatpak/flatpak.rst (hunk 2 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: Hold pls @krisdale . @bktan8 please first respond to my comments on #481.

File: source/clear-linux/tutorials/flatpak/flatpak.rst (hunk 1 above)
Comment (krisdale) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: @mvincerx confirmed - watching.

Comment (mvincerx) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: Confirmed original edit. @bktan8 thanks for your response. 
@krisdale please revise command to:  `sudo swupd bundle-add desktop-autostart.`

File: source/clear-linux/tutorials/flatpak/flatpak.rst (hunk 2 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: @bktan8 thanks for your response. 
@krisdale please revise  command to reflect:  
`sudo swupd bundle-add desktop-autostart`.
Revise text to reflect above change, pls.
Revise first sentence of note: Flatpak is included via `desktop`, which is included in `desktop-autostart`.

File: source/clear-linux/tutorials/flatpak/flatpak.rst (hunk 1 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/flatpak/flatpak.rst]: Confirmed my original comment above: Please change `desktop` to `desktop-autostart`.

**Comment by MCamp859** [comment]
Closing this PR in favor of #753 (same content, updated for new org)

**Pull request closed without merging**